pcsx2-qt/Settings/GraphicsSettingsWidget.ui

```diff
@@ -55,7 +55,7 @@
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
-      <number>0</number>
+      <number>9</number>
```

Comment:
Please revert this back to 0, otherwise the graphics section would default to the advanced tab upon opening the setting.